internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -208,7 +208,6 @@ func (r *ClusterDeploymentsReconciler) updateClusterMetadata(ctx context.Context
 	cluster.Spec.Installed = true
 	cluster.Spec.ClusterMetadata = &hivev1.ClusterMetadata{
 		ClusterID: c.OpenshiftClusterID.String(),
-		InfraID:   c.ID.String(),
```

Comment:
So what value would it have now? Would it remain empty? If so, so why does it exist?

Reply:
It's a fairly short ID usually used to tag resources in a cloud account.
We should not use it to expose internal UUIDs

Reply:
It will remain empty in our flow. It is in used with other hive installation flows.